tcl/html.tcl

```diff
@@ -60,6 +64,46 @@ proc qc::html_tag { tagName args } {
     }
 }
 
+proc qc::h_tag { tag_name args } {
```

Comment:
Yeah, I think that's the case.

These were pulled over from mla-www and I can't find any documentation. I'll add docs for `qc::h`.